db.json

```diff
@@ -5523,5 +5523,39 @@
                 }
             }
         }
+    },
+    "SceUsbSer":{
+	"nid": 4175008527,
+	"modules": {
+	    "SceUsbSerialForDriver": {
+		"kernel": true,
+		"nid": 2284438259,
+		"variables": {},
+		"functions":{
+			"SceUsbSerialStartForDriver": 4276614330,
+			"SceUsbSerialSetupForDriver": 1493929879,
+			"SceUsbSerialStatusForDriver": 2353162502,
+			"SceUsbSerialCloseForDriver": 3870418511,
+			"SceUsbSerialget_lenForDriver": 4113675694,
+			"SceUsbSerialSendForDriver": 204370880,
+			"SceUsbSerialrecvForDriver": 1801333103
```

Comment:
we don't know original sdk rules, and db file just defined function-name map.
so we can use own naming rule (well... maybe related #80)

current project convention is function name should be `camelCase`.
i guess, you should change function name in both db & header  :)